### ISSUE TITLE
Install not working on MacOS 11.2.1

### ISSUE THREAD

**Issue by yves-surrel** [vincefn/pyvkfft#3]
Hi

Seems very exciting, but

```
pip install pyvkfft 
```

returns

```
clang: error: invalid argument '-bundle' not allowed with '-dynamiclib'
```

**Comment by vincefn** [comment]
I've only tested on macOS 10.13.6 (clang with Xcode 9.2).

Can you try after removing the --shared option in setup.py ?

**Comment by yves-surrel** [comment]
@vincefn  As you suggested, I changed
```
        extra_link_args = ['-Wl,-framework,OpenCL', '--shared']
```
to
```
        extra_link_args = ['-Wl,-framework,OpenCL']
```
at line 70 (in `locate_opencl`).

Fixed, thx

**Comment by tlambert03** [comment]
I can also confirm on both macos11 and 10.15 that removing `--shared` is necessary to build.

**Comment by vincefn** [comment]
Fixed by e843fdd443. The `--shared` may still be necessary on older Xcode/llvm compilers
